### ISSUE TITLE
Ability so setup a dynamic link to the svg file.

### ISSUE THREAD

**Issue by bakdauletkarakulov** [okviz/synoptic-panel-issues#18]
### The feature request

We would like to use Synoptic panel visual inside the generic file. Let's assume we have 100 different customers, and 100 different svg maps. Inside the generic file we would like to only provide svg file link(column in table) and based on that link the visual will show up the map of customer. 

### Proposed solution

We will be able to use the visual inside generic file and avoid manual selection of file.

**Comment by danieleperilli** [comment]
This is something we are considering for the next version of Synoptic Panel (Paid) that we will release by the end of Q2. Thanks.

**Comment by chebyshevy** [comment]
> This is something we are considering for the next version of Synoptic Panel (Paid) that we will release by the end of Q2. Thanks.

Hi, Has this feature been released please? I have some svg images which have been saved via URL and I would like to use the Synoptic Panel visual to visualize these images

**Comment by danieleperilli** [comment]
Yes, it is would be called auto fetch. In a nutshell: if your map has several areas with different ids and you have several maps in the same location named with the area id, Synoptic Panel will allow you to drill down to each map when you click on the area. This feature will be included in the next version of the visual. Unfortunately, there have been delays, so we have rescheduled the release to the end of October 2024.

**Comment by danieleperilli** [comment]
The new version is out: [https://okviz.com/synoptic-panel](https://okviz.com/synoptic-panel/)
